.husky/commit-msg

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn commitlint -E HUSKY_GIT_PARAMS
```

Comment:
This was failing, see: https://github.com/typicode/husky/issues/779#issuecomment-713207753